### PR TITLE
Make Dockerfile compatible with PHPUnit 9

### PR DESCRIPTION
**Pull request by stephandesouza**
PHPUnit 9 only runs with PHP >=7.3, also updated to Composer 2

### CODE REVIEW

**Comment by Smolevich** [comment]
ARG is param can be passed through `docker build --build-arg PHP_VERSION=8.0 ...` I think that changing default params is can be realized later, after big refactoring code base

**Comment by stephandesouza** [comment]
Oh, thanks!! I really forgot about `build-args` on Docker.

**Comment by divine** [comment]
Docker file already doesn't run correctly tests. Bumping two variables doesn't change anything.

Thanks!

**Comment by stephandesouza** [comment]
Well, ok, but I could run standalone test correctly on master with this bump.

So others might run this kind of tests too.

**Comment by divine** [comment]
> Well, ok, but I could run standalone test correctly on master with this bump.
> 
> So others might run this kind of tests too.

Docker file doesn't contain any code to run tests. Could you please show me where exactly it does otherwise?

Thanks!

**Comment by stephandesouza** [comment]
Sure, with docker-compose it runs smoothly: `docker-compose run tests ./vendor/bin/phpunit`
image

**Comment by divine** [comment]
> Sure, with docker-compose it runs smoothly: `docker-compose run tests ./vendor/bin/phpunit`
> image

What was I meaning that when you start it with docker-compose the PHP container should run tests automatically.

There is no point in running it manually each time. @Smolevich what do you think?

Thanks!

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG PHP_VERSION=7.2
-ARG COMPOSER_VERSION=1.8
+ARG PHP_VERSION=7.3
```
Comment (divine) [inline Dockerfile:1]:
```suggestion
ARG PHP_VERSION=7.4
```
7.4 is currently active version

**Review by divine — APPROVED** [review]

**Comment by divine** [comment]
Thank you for your contribution! Much appreciated!

**Pull request merged**